Mathlib/RingTheory/PowerBasis.lean

```diff
@@ -143,6 +144,25 @@ theorem algHom_ext {S' : Type*} [Semiring S'] [Algebra R S'] (pb : PowerBasis R
   rw [← Polynomial.aeval_algHom_apply, ← Polynomial.aeval_algHom_apply, h]
 #align power_basis.alg_hom_ext PowerBasis.algHom_ext
 
+open Ideal Finset Submodule in
+theorem exists_sModEq (pb : PowerBasis A B) (b : B) :
```

Comment:
Can you provide a version with `∃ a, pb.gen ∣ b - algebraMap A B a`?

Reply:
Done, thanks!